bitnami/contour-operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,208 @@
+{{- if .Values.rbac.create }}
+apiVersion: {{ include "common.capabilities.rbac.apiVersion" . }}
+kind: ClusterRole
+metadata:  
```

Comment:
Trailing spaces here. Use one of the IDE plugins that trims trailing whitespaces